test/shopify_app/controller_concerns/login_protection_test.rb

```diff
@@ -240,8 +240,10 @@ class LoginProtectionControllerTest < ActionController::TestCase
     end
   end
 
-  test '#activate_shopify_session with Shopify session, clears top-level auth cookie' do
+  test '#activate_shopify_session with only shop_session, clears top-level auth cookie' do
     with_application_test_routes do
+      ShopifyApp::SessionRepository.user_storage = nil
```

Comment:
Do these need to get reset after the test completes?

Reply:
I believe they're reset in the `setup` method of this test file